filters/ThirdParty/filter_251_FutaFilterPrivacy/metadata.json

```diff
@@ -0,0 +1,20 @@
+{
+  "filterId": 250,
+  "name": "FutaFilter URL Privacy Guardner List",
+  "description": "Now the tracker is proliferating and can be seen everywhere. This list designed to against URL tracks, protect users privacy.",
+  "timeAdded": 1616091726943,
+  "homepage": "https://github.com/FutaGuard/FutaFilter/blob/master/README.en.md",
```

Comment:
@tasi788

Could you please supply a separate homepage just for removeparam list?

This filter list does not seem to be specific to Taiwanese websites so that would be rather strange when people will get to that homepage and see an unrelated description.

Reply:
sure, I can make a homepage for removeparam list ASAP.